radixdlt/babylon-wallet-android
# [ABW-3316] Update gateway to 1.5.1

## Description
I have done some basic testing and it appear to work fine
- used schema from 1.5.2, according to slack thread on the gateway release they are the same
- moved some classes that we defined manually (from coreAPI) out of generated file to avoid re-adding them manually on subsequent gateway updates

## How to test

1. Run app on stokenet, which was wiped out.
2. Verify if core app features works - wallet screen/details/transfer/integration tests

## PR submission checklist
- [x] I have tested app on stokenet
